server/auth/github/client_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !go1.10
```

Comment:
@ejholmes any clue what this does? I copied it from `internal-v2` and now the tests pass?!